rust-toolchain

```diff
@@ -1,14 +1,13 @@
-# If you see this, run `rustup self update` to get rustup 1.23 or newer.
+# If you see this, run `rustup self update` to get rustup 1.23 or newer
 
-# NOTE: above comment is for older `rustup` (before TOML support was added),
+# NOTE: above comment is for older `rustup` (before TOML support was added)
 # which will treat the first line as the toolchain name, and therefore show it
-# to the user in the error, instead of "error: invalid channel name '[toolchain]'".
+# to the user in the error, instead of "error: invalid channel name '[toolchain]'"
 
 [toolchain]
-channel = "nightly-2022-12-18"
+channel = "nightly-2023-01-21"
 components = ["rust-src", "rustc-dev", "llvm-tools-preview"]
-# commit_hash = 0468a00ae3fd6ef1a6a0f9eaf637d7aa9e604acc
+# commit_hash = 5ce39f42bd2c8bca9c570f0560ebe1fce4eddb14
 
 # Whenever changing the nightly channel, update the commit hash above, and make
-# sure to change REQUIRED_TOOLCHAIN in crates/rustc_codegen_spirv/src/build.rs also.
-
+# sure to change REQUIRED_TOOLCHAIN in crates/rustc_codegen_spirv/src/build.rs also
```

Comment:
I'm confused, are the punctuation changes here intentional? The removal of a `,`, not just `.` makes it feel accidental.

Reply:
Ugh. This is VSCode keeping thinking that that file is markdown instead of toml. I thought I disabled format-on-write though, but I definitely not changed those things by hand :)